specs/IsSmartScreenEnabled

```diff
@@ -0,0 +1,88 @@
+# Background
+
+[Edge SmartScreen](https://docs.microsoft.com/en-us/deployedge/microsoft-edge-security-smartscreen) helps you identify reported phishing and malware websites, and also helps you make informed decisions about downloads.
+
+Currently, users can use `options->put_AdditionalBrowserArguments(L"--disable-features=msSmartScreenProtection")` to control SmartScreen in the WebView2 application. It is essentially a startup parameter of the browser process. It must be determined when the WebView2Environment is created. And Cannot be modified at runtime.
+
+To support more flexibility. We introduce a new API.
+
+In this document we describe the new setting. We'd appreciate your feedback.
+
+
+# Description
+Users can use ICoreWebView2Environment to control it.When it is changed, all WebView2 applications using the same path of the user data folder will be changed.
```

Comment:
> allWebView2 applications using the same path of the user data folder will be changed.

If two different host apps connect to the same browser process / user data folder, and one changes the IsSmartScreenEnabled property value, does the other one see the now updated value? I guess that's more of an implementation question.